app/src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,78 @@
+create table if not exists users
+(
+    user_id    bigint       not null primary key,
+    username   varchar(25)  not null,
+    password   varchar(255) not null,
+    user_type  varchar(16)  not null check (user_type in ('ROLE_USER', 'ROLE_ADMIN')),
+    created_at TIMESTAMP(6) not null,
+    updated_at TIMESTAMP(6) not null
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8mb3;
+
+create table if not exists product
+(
+    product_id  bigint       not null primary key,
+    name        varchar(255) not null,
+    description longtext     not null,
+    price       varchar(255) not null,
+    quantity    bigint       not null,
+    start_time  TIMESTAMP(6) not null,
+    end_time    TIMESTAMP(6) not null,
+    created_at  TIMESTAMP(6) not null,
+    updated_at  TIMESTAMP(6) not null
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8mb3;
+
+create table if not exists cart
+(
+    cart_id    bigint       not null primary key,
+    user_id    bigint       not null unique,
+    created_at TIMESTAMP(6) not null,
+    updated_at TIMESTAMP(6) not null
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8mb3;
+
+create table if not exists cart_product
+(
+    cart_id    bigint not null references cart (cart_id),
+    product_id bigint not null references product (product_id),
+    quantity   bigint,
```

Comment:
여기 not null도 필요할거 같아요
```suggestion
    quantity   bigint not null,
```